### DISCUSSION TITLE
Video DownloadHelper icon always disable on Youtube upon Surfshark VPN connected

### DISCUSSION THREAD

**Discussion by rickygai** [aclap-dev/video-downloadhelper#1093] [Help and Q&A]
### Your question

**Microsoft Windows Version 22H2 (OS Build 22621.3958)
Microsoft Edge Version 127.0.2651.86 (Official build) (64-bit)
Video DownloadHelper for Edge Version 9.0.2.12
Surfshark VPN v5.8.2**

After connected to Surfshark VPN, the Video DownloadHelper icon on the toolbar totally disabled upon any Youtube videos played.

Please advise.

### Addon & CoApp version and details
Video DownloadHelper for Edge Version 9.0.2.12

**Comment by Mattmjs** [comment]
Do you mean it doesn't detect anything with the
vpn enabled. Is it the full desktop vpn and not a
vpn extension ? Vpn extensions can cause problems. Also are you using YouTube premium?
This is another thing that has caused issues.

**Reply by rickygai** [reply]
hi,

_"Is it the full desktop vpn"_ 
Yes, it is a full desktop vpn application.

_"Also are you using YouTube premium?"_
No, but other Youtube downloader software like YouTubeByClick able to download the same video upon desktop VPN enabled.

**Comment by paulrouget** [comment]
Yeah… it's an ongoing issue with Youtube: https://github.com/aclap-dev/video-downloadhelper/discussions/1074 - I'm working on it.

**Reply by rickygai** [reply]
> Yeah… it's an ongoing issue with Youtube: #1074 - I'm working on it.
Thank you :)
